src/test/resources/dialogmelding_dialog_svar_innkalling_dialogmote.xml

```diff
@@ -81,31 +80,61 @@
                 <Sex DN="Mann" V="1" />
                 <Ident>
                     <Id>3143242342</Id>
-                    <TypeId DN="F�dselsnummer" S="2.16.578.1.12.4.1.1.8116" V="FNR" />
+                    <TypeId DN="Fødselsnummer" S="2.16.578.1.12.4.1.1.8116" V="FNR" />
                 </Ident>
                 <Address>
-                    <Type DN="Postadresse" V="PST" />
+                    <Type DN="Folkeregisteradresse" V="HP"/>
                     <StreetAdr>Sannergata 2</StreetAdr>
                     <PostalCode>0655</PostalCode>
                     <City>OSLO</City>
-                    <County DN="OSLO" V="0712" />
                 </Address>
+                <TeleCom>
+                    <TypeTelecom DN="MobilTelefon" V="MC"/>
+                    <TeleAddress V="tel:12345678"/>
+                </TeleCom>
+                <TeleCom>
+                    <TypeTelecom DN="Hovedtelefon" V="HP"/>
+                    <TeleAddress V="tel:23456789"/>
+                </TeleCom>
             </Patient>
         </MsgInfo>
         <Document>
             <RefDoc>
-                <MsgType DN="XML-instans" V="XML" />
+                <IssueDate V="2020-09-21T21:11:26"/>
+                <MsgType DN="XML-instans" V="XML"/>
                 <Content>
-                    <Dialogmelding xmlns="http://www.kith.no/xmlstds/dialog/2006-10-11"
-                                   xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+                    <Dialogmelding
+                            xmlns="http://www.kith.no/xmlstds/dialog/2006-10-11" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.kith.no/xmlstds/dialog/2006-10-11 dialogmelding.xsd">
                         <Notat>
-                            <TemaKodet DN="Henvendelse om sykefrav�rsoppf�lging" OT="Benyttes n�r henvendelsen gjelder en pasient som er sykmeldt, evt henvendelser knyttet til et tidligere sykefrav�rstilfelle." S="2.16.578.1.12.4.1.1.8128" V="1" />
-                            <TekstNotatInnhold xsi:type="xsd:string">Jeg er klar 12.30. Mvh, Lege legsesn</TekstNotatInnhold>
+                            <TemaKodet DN="Ja, jeg kommer" S="2.16.578.1.12.4.1.1.8126" V="1"/>
```

Comment:
Dette er jo faktisk riktig kodeverk verdi for svar på inkalling til dialogmøte, i forhold til det det var før..